obal/data/modules/changelog.py

```diff
@@ -50,22 +50,27 @@ def main():
 
     changed = False
 
-    if not changelog.startswith('-'):
-        changelog = '- ' + changelog
+    if entry is not None and not entry.startswith('-'):
```

Comment:
It's good enough to check if it's trueish. This also catches empty strings but that's ok because an empty string never starts with `-`.
```suggestion
    if entry and not entry.startswith('-'):
```